server/models/polls.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const PollSchema = new Schema({
+	title: {
+		type: String,
+		required: true,
+	},
+	userId: {
+		type: Number,
```

Comment:
The userId type should match the user collection. There's no _id in the current schema, so my guess is that mongo will create the _id automatically using an ObjectId type, so we should use that here, the other option would be String.

Reply:
MongoDB will automatically create an _id, and so this should be a ObjectId. In addition it should be indexed (https://mongoosejs.com/docs/guide.html#indexes)

Reply:
I believe the type for `userId` should be a `String`. 

When I tried to make a POST request to `/polls/add` on Postman, it returned an instance of a `Poll` but it didn't have a `userId` + it didn't save to the DB.

It gave me this message on the server log:
`ValidationError: polls validation failed: userId: Cast to Number failed for value "5ec5728a07c3da5b8736a32c" at path "userId"`

Changing it to `String` fixed it